airbyte-integrations/connectors/source-tiktok-marketing/manifest.yaml

```diff
@@ -1518,7 +1518,24 @@ definitions:
           "paid_engagement_engaged_view",
           "engaged_view_15s",
           "paid_engaged_view_15s",
-          "paid_engagement_engaged_view_15s"
+          "paid_engagement_engaged_view_15s",
+          "skan_result",
+          "skan_cost_per_result",
+          "skan_conversion",
+          "skan_cost_per_conversion",
+          "skan_click_time_conversion", 
+          "skan_click_time_cost_per_conversion",
+          "skan_click_time_conversion_rate",
+          "skan_app_install",
+          "skan_registration",
+          "skan_complete_tutorial",
+          "skan_purchase",
+          "skan_cost_per_purchase",
+          "skan_purchase_rate",
+          "skan_total_purchase",
+          "skan_cost_per_total_purchase",
+          "skan_total_purchase_value",
+          "skan_total_repetitive_active_pay_roas",
```

Comment:
have you tested this? 
just got an error `Invalid value for dimensions: hourly breakdown is not supported for skan metrics.` while running this stream locally